tests/check-entries-csv

```diff
@@ -261,46 +261,46 @@ def check_column_birthyear(s, row, csv):
             pwarning("BirthYear looks implausible: '%s'" % s)
 
 
-def check_column_weightclass(s, row, csv):
-    if s[-1] is '+':
-        check_positive_number_column(s[:-1], row, csv)
-
-    ipf_men = ['53', '59', '66', '74', '83', '93', '105', '120', '120+']
-    ipf_women = ['43', '47', '52', '57', '63', '72', '84', '84+']
-    ipf_women_extra = ['30', '35', '40']
-    ipl = ['48', '52', '56', '60', '67.5', '75', '82.5',
-           '90', '90+', '100', '110', '125', '125+']
-    rum_extra = ['44', '140', '140+']
-    parapl_men = ['49', '54', '59', '65', '72', '80', '88', '97', '107', '107+']
-    parapl_women = ['41', '45', '50', '55', '61', '67', '73', '79', '86', '86+']
-
-    apa_extra = ['39']  # Youth.
-    proraw_extra = ['70', '70+', '75+', '80', '95', '110+']
-    rps_extra = ['155']
-    spf_extra = ['117.5', '100+', '67.5+']
-    idfpf_extra = ['47.5', '50.5', '55.5', '58.5']
-    ipf_extra = ['60+']
-    old_usapl = ['145', '145+']
-    wnpf_extra = ['34']
-    wpau_extra = ['155', '155+']
-
-    ffforce_extra = ['83+']  # Cadet division, for youth.
-    parapl_extra = ['82.5+']
-
-    classes = ipf_men + ipf_women + ipl + rum_extra + proraw_extra + \
-        rps_extra + spf_extra + ipf_extra + ipf_women_extra + \
-        old_usapl + ffforce_extra + wnpf_extra + apa_extra + \
-        idfpf_extra + parapl_extra + wpau_extra + parapl_men + parapl_women
-
-    if s not in classes:
-        pwarning("Unknown weight class: %s" % s)
+def check_row_weightclass(row, csv, weightclasses):
+    if 'WeightClassKg' in csv.fieldnames and row[csv.index('WeightClassKg')] != "":
```

Comment:
Reduce the level of indentation by changing this to a guard:

```
if 'WeightClassKg' not in csv.fieldnames or row[csv.index('WeightClassKg')] == '':
    return
```

Guards are always preferred over indentation.